src/sentry/static/sentry/app/views/performance/table.tsx

```diff
@@ -255,4 +279,46 @@ class Table extends React.Component<Props, State> {
   }
 }
 
+type KeyTransactionActionProps = {
+  projectID: number;
+  organization: Organization;
+  transactionName: string;
+};
+
+function KeyTransactionAction(props: KeyTransactionActionProps) {
+  const {projectID, organization, transactionName, ...rest} = props;
+  return (
+    <KeyTransactionQuery
```

Comment:
This loads/reloads every time the cell actions are open?

Reply:
Yes, there is no nice place to store this information at the moment. In order to do this only once, `CellAction` needs to be reworked quite a bit as it was not initially made for stateful actions.

Reply:
👍